nginx.conf

```diff
@@ -21,8 +21,8 @@ http {
 			proxy_http_version 1.1;
 			proxy_set_header Connection "";
 
-			auth_basic "Restricted"; #For Basic Auth
-			auth_basic_user_file /etc/nginx/conf.d/.htpasswd; #For Basic Auth
+			# auth_basic "Restricted"; #For Basic Auth
+			# auth_basic_user_file /etc/nginx/conf.d/.htpasswd; #For Basic Auth
```

Comment:
Why are the basic auth settings commented out?

Reply:
That is password-based authentication. Originally, I used nginx to do authentication. Now the ELK setup uses a security plugin from Open Distro for Elastic Search.

Reply:
Okay, sounds good.  Since it's commented out, maybe a comment explaining this would be useful here?